### ISSUE TITLE
Fails to start, due to grave error

### ISSUE THREAD

**Issue by hwansing** [pastillilabs/packages#1]
Just updated to v.279, app cannot start, an popup comes up saying something like "A grave error occured, application will close". Rebooting the device did not help. This is on Fairphone 3+ with latest OS version.
![Screenshot_20210927-071535_Situations](https://user-images.githubusercontent.com/8604846/134850041-13678cfb-1684-451a-9a25-9e2344bbecc3.png)

**Comment by hhaveri** [comment]
Thanks for the report! Did you install from Play store or from the github packages? if you did install from Play store, can you please try the github packages instead: https://github.com/pastillilabs/packages/releases/tag/279

**Comment by hwansing** [comment]
Hi,

Am 27. September 2021 11:43:23 MESZ schrieb Heikki Haveri ***@***.***>:
>Thanks for the report! Did you install from Play store or from the github packages? if you did install from Play store, can you please try the github packages instead: https://github.com/pastillilabs/packages/releases/tag/279

I installed from the app store (this is not Google Play Store; Fairphone is running
a Google-free "android" clone similar to Lineage OS and has its own app store).

Now I installed 279 from github and it works. 
Huuh? I did not imagine this would be possible, but ... it works now!
Thanks

Feel free to close this issue


Holger




-- 
Sent from /e/ OS on Fairphone3
